projects/typescript-vue/src/app.vue

```diff
@@ -0,0 +1,212 @@
+<script setup lang="ts">
+import { onMounted, ref } from 'vue';
+import { BpmnElement, BpmnElementsRegistry, BpmnVisualization, FitType, OverlayPosition, ShapeBpmnElementKind } from 'bpmn-visualization';
+
+let vis: BpmnVisualization;
+let registry: BpmnElementsRegistry;
+const allFlowNodes = ref<BpmnElement[]>([]);
+const selectedElement = ref<string>();
+const dropdownActive = ref(false);
+const loading = ref(true);
+const loadError = ref();
+
+const nodeDropdown = ref<string>();
+const overlayText = ref<string>('');
+const overlayPosition = ref<OverlayPosition>('top-left');
+
+
+onMounted(async () => {
+    const res = await fetch('/pizza-collaboration.bpmn');
```

Comment:
**nitpick:** the diagram is not loaded work when the application is not deployed in the root context. The fetch call returns an HTTP 404 error and the loading spin is displayed forever.
You can reproduce it by downloading the artifact produced by the GitHub Action or by building the app locally, then by running a webserver serving the parent of the folder storing the application.
We need it to work like this as the application is deployed in the PR preview environment (and in the future on GH Pages) with the other projects, so it will be served from its own context, for example from http://127.0.0.1:8080/release/typescript-vue/


Notice that because of this error, it is not possible to load local file. An internal error occurs
![image](https://github.com/process-analytics/bpmn-visualization-examples/assets/27200110/b9c50ba8-6b2f-4394-b2a6-a4b638f35088)

<details>
<summary>Console error</summary>
<pre>
07:18:55,588 TypeError: i is undefined
    m http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:29
    onload http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:29
    v http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:29
    callWithErrorHandling http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    callWithAsyncErrorHandling http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    i http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    addEventListener http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    patchEvent http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    patchProp http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    T http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    O http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    x http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    P http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    T http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    O http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    x http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    P http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    T http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    O http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    x http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    P http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    T http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    O http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    x http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    P http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    k http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    x http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    H http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    run http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    update http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    J http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    z http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    G http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    x http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    mt http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    mount http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    mount http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:1
    <anonymous> http://127.0.0.1:8080/all-typescript-projects-65bdd4b2d2ebdc4304837200f78c263428c68858/release/typescript-vue/assets/index-be7e40a7.js:29
index-be7e40a7.js:29:41025
</pre>
</details>